proto/sensr_proto/type.proto

```diff
@@ -63,3 +63,12 @@ message Object {
   History history = 102;
   Prediction prediction = 103;
 }
+
+message PredictedReachableSet {
```

Comment:
AFAIK, J made Prediction to share this info too.
Could you integrate PredictedReachableSet into Prediction.